src/microbit/test/test_image.py

```diff
@@ -269,3 +270,15 @@ def test_str(self, image, repr_actual, str_actual):
         str_output = str(image)
         assert repr_actual == repr_output
         assert str_actual == str_output
+
+    @pytest.mark.parametrize(
+        "const, actual",
+        [
+            (Image.SNAKE, Image("99000:99099:09090:09990:00000:")),
+            (Image.PITCHFORK, Image("90909:90909:99999:00900:00900:")),

```

Comment:
I would probably use `Image(CONSTANTS.IMAGE_PATTERN[pattern])`, in case a pattern gets changed in the future.

Reply:
The point of using Image.SNAKE + etc. is because Image.SNAKE should specifically return a read-only instance of an image. Image(CONSTANTS.IMAGE_PATTERN[pattern]) would create it naturally (not read-only)

Reply:
Ah my bad, I meant for the `actual` variable, not the `const` variable.

Reply:
ah ok! I'll add that in